pkg/server/runner.go

```diff
@@ -136,6 +139,20 @@ func (r *Server) Flags(cmd *cli.Command) {
 			Destination: &r.EnableTls,
 			Value:       r.EnableTls,
 		},
+		&cli.StringFlag{
+			Name:        "builtin-catalog-provider",
+			Usage:       "Builtin catalog provider, Github or Gitee.",
+			Destination: &r.BuiltinCatalogProvider,
+			Value:       r.BuiltinCatalogProvider,
+			Action: func(c *cli.Context, s string) error {
+				switch s {
+				case types.GitDriverGitee, types.GitDriverGithub:
```

Comment:
Should we make it case insensitive in cli args?

Reply:
Updated, it's value is case-insensitive now.